--- a/dist/index.js
+++ b/dist/index.js
@@ -1360,6 +1360,7 @@
       if (!done) {
         await exitPromise.catch(() => {
         });
+        await new Promise((resolve) => setTimeout(resolve, 250));
       }
       return done ?? {
         success: false,
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -1362,6 +1362,7 @@
       if (!done) {
         await exitPromise.catch(() => {
         });
+        await new Promise((resolve) => setTimeout(resolve, 250));
       }
       return done ?? {
         success: false,
